pkg/apis/kubeflow/v1/types.go

```diff
@@ -37,6 +37,14 @@ type MPIJobList struct {
 	Items           []MPIJob `json:"items"`
 }
 
+// ElasticPolicy describes which Elastic Mode or none is enabled.
+type ElasticPolicy string
+
+const (
+	ElasticPolicyUndefined ElasticPolicy = ""
+	ElasticPolicyHorovod ElasticPolicy = "ElasticHorovod"
```

Comment:
Can we detect this automatically?

Reply:
I think not. The only difference lies on the launch command, which can be either specified or coded into the container image.

Btw, the elastic mode seems compatible with non-elastic cases.

Reply:
That's good to know.